src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -158,7 +158,18 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({
     refetchQueries: [{ query: SEARCH_QUERY, variables: SEARCH_QUERY_VARIABLES }],
   });
 
-  // this function is called when the message is sent
+  const updatePayload = (payload: any, selectedTemplate: any, variableParam: any) => {
+    // add additional param for template
+    if (selectedTemplate) {
+      /* eslint-disable no-param-reassign */
+      payload.isHsm = selectedTemplate.isHsm;
```

Comment:
We should not update the function parameters. Instead, we can create another variable that can be updated.